brewtils/rest/client.py

```diff
@@ -943,3 +944,74 @@ def get_tokens(self, username=None, password=None):
             self.session.headers["Authorization"] = "Bearer " + self.access_token
 
         return response
+
+    @enable_auth
+    def get_topic(self, topic_name, **kwargs):
+        # type: (str, **Any) -> Response
+        """Performs a GET on the Topic URL
+
+        Args:
+            topic_name: Topic Name
+            **kwargs: Query parameters to be used in the GET request
+
+        Returns:
+            Requests Response object
+        """
+        return self.session.get(self.topic_url + quote(topic_name), params=kwargs)
+
+    @enable_auth
+    def get_topics(self, **kwargs):
```

Comment:
Is the API going to support kwargs?

Reply:
Not at this time. Removed kwargs from get_topics.